docs/UserGuide.md

```diff
@@ -744,13 +800,20 @@ Step by step guide:
            
        Event is not found in the list.
        
-1. If there are no students' data in the student list:
+1. If there is no students' data in the student list:
```

Comment:
Perhaps you can change the phrasing to "If the student list collection is empty:  "

Reply:
ok